recipes/diffutils/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "diffutils" %}
+{% set version = "3.6" %}
+{% set sha256 = "d621e8bdd4b573918c8145f7ae61817d1be9deb4c8d2328a65cea8e11d783bd6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+test:
+  commands:
+    - cmp --version
+    - diff --version
+    - diff3 --version
+    - sdiff --version
+
+about:
+  home: https://www.gnu.org/software/diffutils/
+  license: GPL-3.0-or-later
+  license_family: GPL
+  license_file: COPYING
+  summary: 'GNU Diffutils is a package of several programs related to finding differences between files.'
+
+  doc_url: https://www.gnu.org/software/diffutils/manual/html_node/index.html
+  dev_url: http://git.savannah.gnu.org/cgit/diffutils.git
+
+extra:
+  recipe-maintainers:
+    - bgruening
+    - beatrizserrano
```

Comment:
@beatrizserrano do you agree to be a maintainer of this recipe?

Reply:
sure :)